packages/analytics/shippers/elastic_v3/browser/src/browser_shipper.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  BehaviorSubject,
+  interval,
+  Subject,
+  bufferWhen,
+  exhaustMap,
+  filter,
+  skipWhile,
+} from 'rxjs';
+import type {
+  AnalyticsClientInitContext,
+  Event,
+  EventContext,
+  IShipper,
+  TelemetryCounter,
+} from '@kbn/analytics-client';
+import type { ElasticV3ShipperOptions } from '@kbn/analytics-shippers-elastic-v3-common';
+import {
+  buildUrl,
+  createTelemetryCounterHelper,
+  eventsToNDJSON,
+} from '@kbn/analytics-shippers-elastic-v3-common';
+
+export class ElasticV3BrowserShipper implements IShipper {
+  public static shipperName = 'elastic_v3_browser';
```

Comment:
that's a good point! Having the same name will make sure we don't register both in the same environment. However, I thought about the implications it may have on the `telemetryCounters$`. Having different names makes it easier to figure out which one is dropping/failing to send requests. 

In Kibana we will know because they'll likely be reported as UI vs. Usage Counters. But it's probably easier to differentiate if it goes down to the shipper name. What do you think?

Reply:
If it's easier to differentiate them using distinct names, then I think it's fine to do so.